technical-resources/admin-functions/operational-admin-actions.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+The operationalAdmin complements the Governor's capabilities by having the authority to execute essential operational functions for routine protocol management. The role holds limited powers compared to the Governor, ensuring a balance between operational efficiency and security.
+
+The Governor retains the exclusive authority to appoint or replace the operationalAdmin at any time.
+
+# Operational Admin Functions
+
+## Globals
+
+* `activatePoolManager`
+* `setBootstrapMint`
+* `setCanDeployFrom`
+* `setMaxCoverLiquidationPercent`
+* `setMinCoverAmount`
+* `setPlatformManagementFeeRate`
+* `setPlatformOriginationFeeRate`
+* `setPlatformServiceFeeRate`
+* `setValidInstanceOf`
+* `setValidPoolDelegate`
+* `setValidBorrower`
+
+## Pool Manager
+
+* `finishCollateralLiquidation`
+* `setPendingPoolDelegate`
+* `triggerDefault`
+
+## Withdrawal Manager
```

Comment:
Specify it's the cyclical withdrawal manager and add a section for the queued one as well.

Reply:
added ppm too, should we update wiki Globals Update ‐ Q4 Release ?

Reply:
No need. The wiki will be deleted after the release